packages/connect-thegraph-voting/src/entities/Vote.ts

```diff
@@ -22,7 +21,6 @@ export default class Vote extends Entity implements VoteData {
   readonly id!: string
   readonly creator!: string
   readonly metadata!: string
-  readonly open!: boolean
```

Comment:
I realized that the subgraph is not really capable of tracking this property since its not triggered by an event, but by a time out. There should be a way to determine it from connect, but I just disabled it for now.